modules/cudf/src/series.ts

```diff
@@ -419,6 +419,28 @@ export class AbstractSeries<T extends DataType = any> {
     }
   }
 
+  /**
+   * Concat a Series to the end of the caller, returning a new Series.
+   *
+   * @param other The Series to concat to the end of the caller.
+   *
+   * @example
+   * ```typescript
+   * import {Series} from '@rapidsai/cudf';
+   *
+   * // Float64Series
+   * Series.new([1, 2, 3]).concat(Series.new([4, 5, 6])) // [1, 2, 3, 4, 5, 6]
+   * // StringSeries
+   * Series.new(["foo"]).concat(Series.new(["bar"])) // ["foo", "bar"]
+   *
+   * ```
+   */
+  concat(other: Series<T>, memoryResource?: MemoryResource): Series<T>;
+
+  concat(other: any, memoryResource?: MemoryResource): Series<T> {
```

Comment:
Why is the second `other: any` signature needed?

Reply:
IDE was complaining about passing in `other._col` when the method definition looks like this...
```
  concat(other: Series<T>, memoryResource?: MemoryResource): Series<T> {
    return Series.new(this._col.concat(other._col, memoryResource));
  }
 ```
 
 ```
 Argument of type 'Column<Int64> | Column<Float64> | Column<Bool8> | Column<Int32> | Column<Int8> | Column<Int16> | ... 7 more ... | Column<...>' is not assignable to parameter of type 'Column<T>'.
  Type 'Column<Int64>' is not assignable to type 'Column<T>'.
    Type 'Int64' is not assignable to type 'T'.
      'Int64' is assignable to the constraint of type 'T', but 'T' could be instantiated with a different subtype of constraint 'DataType'.ts(2345)
 ```
 
 It seems like `Int64` is apart of `DataType`, so this is a bit odd...

Reply:
Spoke with @bryevdv a bit about this, ended up with just casting `other._col`
```
return Series.new(this._col.concat(other._col as Column<T>, memoryResource));
```
and removing the `other: any` signature. Let me know if you have any other ideas @trxcllnt , otherwise I'll update to use this.

Reply:
That fix looks correct, unfortunately... :confused: